packages/driver/src/dom/visibility.ts

```diff
@@ -329,7 +343,6 @@ const elIsOutOfBoundsOfAncestorsOverflow = function ($el: JQuery<any>, $ancestor
     return false
   }
 
-  // fix for 29605 - display: contents
```

Comment:
any reason we removed the comment here? I like keeping comments that link back to issues if possible

Reply:
This comment felt superfluous to me since the function is named `elHadDisplayContents` and you can easily determine when the line of code was added. Really we were adding support for `display: contents` instead of fixing an issue with `display: contents`.